src/main/resources/application.yaml

```diff
@@ -80,7 +80,7 @@ govnotify.baseUrl: "https://api.notifications.service.gov.uk"
 notificationSender.deduplicateSendsWithinSeconds: 60
 iaCcdFrontendUrl: ${IA_CCD_FRONTEND_URL:http://localhost:3451/}
 
-iaAipFrontendUrl: ${IA_AIP_FRONTEND_URL:https://localhost:3000/}
+iaAipFrontendUrl: ${IA_AIP_FRONTEND_URL:https://immigration-appeal.platform.hmcts.net/}
```

Comment:
This should be set by setting the IA_AIP_FRONTEND_URL environment property not changing the default.